rollout/controller.go

```diff
@@ -148,6 +151,8 @@ func NewController(cfg ControllerConfig) *Controller {
 		},
 	}
 
+	gvk := schema.ParseGroupResource("virtualservices.networking.istio.io").WithVersion(cfg.DefaultIstioVersion)
```

Comment:
1. The gvk variable is the wrong acronym since its a GroupVersionResource and not GroupVersionKind, so it would be gvr.

2. Consider constructing the GVR explicitly without parsing. i.e.

```golang
	gvr := schema.GroupVersionResource{
		Group:    "networking.istio.io",
		Version:  cfg.DefaultIstioVersion,
		Resource: "virtualservices",
	}
```